src/components/AddTodo.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import './AddTodo.css';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { GrAddCircle } from 'react-icons/gr';
+import PropTypes from 'prop-types';
```

Comment:
- I suggest that you should not disable linters because linters force us to write good code instead try to resolve the error 😊

Reply:
Hi, @sahlminkok  This is an eslint error on form control that doesn't apply on the scope of this project. 

Thanks.